client/src/EstimatesComparison/scheme.js

```diff
@@ -172,7 +173,14 @@ function get_data_by_org(include_stat) {
 
       let current_value = 0;
       if (current_doc_is_mains) {
-        current_value = _.sumBy(rows, "current_value") || 0;
+        // Need to check if all current_values are 0. Inactive if all are 0, else sum of it. Need to do an extra check in case sum=0
+        const org_is_active = !_.every(
+          rows,
+          ({ current_value }) => current_value === 0
+        );
```

Comment:
Not really worth changing, but `_.every` has to check every entry. `_.some` can exit on the first false. So, for a basically meaningless optimization, this is better:
```
_.some(rows, ({ current_value }) => current_value !== 0 )
```